Java/core/src/main/java/com/amazon/randomcutforest/store/RCF3PointStoreLarge.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.randomcutforest.store;
+
+import static com.amazon.randomcutforest.CommonUtils.checkArgument;
+
+import java.util.Arrays;
+
+public class RCF3PointStoreLarge extends RCF3PointStore {
+
+    static int INFEASIBLE_LOCN = (int) -1;
```

Comment:
@sudiptoguha I think this bug https://github.com/aws/random-cut-forest-by-aws/pull/322 is caused by this line, right?

Reply:
It's not a bug. The representation of the pointstore changed :) Look at lines 28 and 31-37; the unused locations have value -1. However for the 2.0.1 models the unused valued could have been anything (the check was via refList). I think 1.3 picked up something earlier than 283.

Reply:
Got it. Thanks, agree this not a bug, just some breaking change compared with RCF jar used by AD 1.3 (PR #282).  Should be ok for `ml-commons` 1.3.1, as it's using PR #309.